kontrakter/politi/varetekt/1.0/readme.md

```diff
@@ -1 +1,24 @@
-## Versjon 1.0 som vi til slutt skal gå i produksjon med i ESAS II
+# Versjon 1.0 som vi til slutt skal gå i produksjon med i ESAS II
+
+## Versjoner log
+### PR ESAS-157 ett schema for begjæring 2
```

Comment:
Blir feil å kalle en changelog readme. Sistnevnte er heller for informasjon om repo/moduler. Man bruker heller `CHANGELOG.md` 😄

Reply:
Keine klokken klingen.